doc/contributing.md

```diff
@@ -16,7 +16,7 @@ What can I do?
 You're in luck. There's various sources for tasks:
 
  - Have a look at the [issue tracker](https://github.com/sfttech/openage/issues), especially issues with labels:
-   - [good first issue](https://github.com/SFTtech/openage/labels/good first issue)!
+   - [good first issue](https://github.com/SFTtech/openage/labels/good%20first%20issue)!
    - [just do it](https://github.com/SFTtech/openage/labels/just%20do%20it)!
```

Comment:
While editing this document, could you also exchange this link with this one, as this states explicitly that is has to be an open issue
`https://github.com/SFTtech/openage/issues?q=is%3Aissue+is%3Aopen+label%3A%22just+do+it%22`

Reply:
Done. 👍